cartographer/io/probability_grid_points_processor.h

```diff
@@ -0,0 +1,56 @@
+#ifndef CARTOGRAPHER_IO_PROBABILITY_GRID_POINTS_PROCESSOR_H_
+#define CARTOGRAPHER_IO_PROBABILITY_GRID_POINTS_PROCESSOR_H_
+
+// Library used for inserting range data points into a probability grid.
+
+#include <memory>
+#include <string>
+
+#include "cartographer/io/file_writer.h"
+#include "cartographer/io/points_batch.h"
+#include "cartographer/io/points_processor.h"
+#include "cartographer/mapping_2d/probability_grid.h"
+#include "cartographer/mapping_2d/proto/range_data_inserter_options.pb.h"
+#include "cartographer/mapping_2d/range_data_inserter.h"
+
+namespace cartographer {
+namespace io {
+
+// Creates a probability grid with the specified 'resolution'.
+// 'range_data_inserter' options are used to configure the range
+// data ray tracing through the probability grid.
```

Comment:
Mention that the `z` component of the data is ignored, i.e. all points are projected into the x-y plane.